backend/btrixcloud/colls.py

```diff
@@ -903,6 +903,18 @@ async def add_successful_crawl_to_collections(
             )
             await self.update_crawl_collections(crawl_id, oid)
 
+    async def purge_dedupe_index(self, coll_id: UUID, org: Organization):
+        """purge dedupe index on collection, raise exception if no index or not ready"""
+        coll = await self.get_collection(coll_id, org.id)
+        if not coll.hasDedupeIndex or not coll.dedupeIndex:
+            raise HTTPException(status_code=400, detail="no_dedupe_index_on_collection")
+
+        if coll.dedupeIndex.state != "ready":
+            raise HTTPException(status_code=400, detail="dedupe_index_not_ready")
+
+        await self.crawl_manager.update_coll_index(coll_id, True)
```

Comment:
```suggestion
        await self.crawl_manager.update_coll_index(coll_id, is_purge=True)
```

Not necessary by any means, but helps a bit with readability not needing to check the other method signature to get the args here